.buildkite/pipeline.yml

```diff
@@ -61,34 +62,20 @@ steps:
             allow_failure: true
           - step: "int-test"
             allow_failure: true
-
-      - label: "E2E Test"
-        key: "e2e-test"
-        command: ".buildkite/scripts/e2e_test.sh"
-        agents:
-          provider: "gcp"
-        depends_on:
-          - step: "unit-test"
-            allow_failure: false
-          - step: "int-test"
-            allow_failure: false
-
-      - label: "Cloud e2e Test"
-        key: "cloud-e2e-test"
-        command: ".buildkite/scripts/cloud_e2e_test.sh"
-        agents:
-          provider: "gcp"
-        depends_on:
-          - step: "unit-test"
-            allow_failure: false
-          - step: "int-test"
-            allow_failure: false
-          - step: "e2e-test"
-            allow_failure: false
```

Comment:
This condition was also added as part of the PR https://github.com/elastic/fleet-server/pull/2646
It needs to be recovered too.

Reply:
restored